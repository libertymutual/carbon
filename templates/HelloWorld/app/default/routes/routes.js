import HomeScreen from '../components/home/HomeScreen';
import ContactScreen from '../components/contact/ContactScreen';
import ProfileScreen from '../components/profile/ProfileScreen';
import AboutScreen from '../components/about/AboutScreen';
import BiometricScreen from '../components/biometric/BiometricScreen';
import LoginScreen from '../components/login/LoginScreen';

/**
 * List of possible navigation routes
 */
const routes = {
  HomeScreen: {
    screen: HomeScreen,
  },
  ContactScreen: {
    screen: ContactScreen,
  },
  ProfileScreen: {
    screen: ProfileScreen,
  },
  AboutScreen: {
    screen: AboutScreen,
  },
  BiometricScreen: {
    screen: BiometricScreen,
  },
  LoginScreen: {
    screen: LoginScreen,
  },
};

export default routes;
