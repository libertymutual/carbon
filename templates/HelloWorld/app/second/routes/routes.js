import HomeScreen from '../components/home/HomeScreen';
import AboutScreen from '../components/about/AboutScreen';

/**
 * List of possible navigation routes
 */
const routes = {
  HomeScreen: {
    screen: HomeScreen,
  },
  AboutScreen: {
    screen: AboutScreen,
  },
};
export default routes;
