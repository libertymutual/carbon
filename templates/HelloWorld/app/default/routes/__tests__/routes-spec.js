import 'react-native';
import React from 'react';
import routes from '../routes';

describe('routes tests', () => {

  test('Routes are valid', () => {
    expect(routes).toMatchSnapshot();
  });
});
