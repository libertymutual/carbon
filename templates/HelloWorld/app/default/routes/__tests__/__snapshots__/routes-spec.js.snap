// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes tests Routes are valid 1`] = `
Object {
  "AboutScreen": Object {
    "screen": [Function],
  },
  "BiometricScreen": Object {
    "screen": [Function],
  },
  "ContactScreen": Object {
    "screen": [Function],
  },
  "HomeScreen": Object {
    "screen": [Function],
  },
  "LoginScreen": Object {
    "screen": [Function],
  },
  "ProfileScreen": Object {
    "screen": [Function],
  },
}
`;
